### PR TITLE
Added Firefox Reality

### PR DESCRIPTION
**Pull request by fernandojsg**
Added on Daydream, Oculus Go and Vive port for Firefox Reality on browser's list
image

Added Firefox Reality on developer section
image

### CODE REVIEW

File: developers/index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h3>
         </h3>
 
         <p>
-          The WebVR API is currently available in <a href="https://webvr.rocks/firefox">Firefox for Windows</a>, in
+          The WebVR API is currently available in <a href="https://webvr.rocks/firefox">Firefox for Windows</a>, in <a href="https://mixedreality.mozilla.org/firefox-reality/">Firefox Reality for Oculus Go, Daydream and Vive Port</a>, in
```
[the remark this answers is not in the document]
Comment (fernandojsg) [reply developers/index.html]: done

File: developers/index.html [hunk 2]
```diff
@@ -61,6 +61,9 @@ <h3>
           <li>
             <a href="https://webvr.rocks/firefox" class="link-important">Download Firefox</a> and <a href="https://nightly.mozilla.org/">Firefox Nightly builds (Vive on macOS)</a> (<a href="https://webvr.rocks/firefox#setup">read instructions</a>)
           </li>
+          <li>
+            <a href="https://mixedreality.mozilla.org/firefox-reality/" class="link-important">Firefox Reality</a> for <a href="https://play.google.com/store/apps/details?id=org.mozilla.vrbrowser">Daydream</a>, <a href="https://www.oculus.com/experiences/go/2208418715853974/">Oculus Go</a> and Vive Port (search <em>'Firefox Reality'</em> on the store).
```
[the remark this answers is not in the document]
Comment (fernandojsg) [reply developers/index.html]: done

File: developers/index.html [hunk 3]
```diff
@@ -115,6 +119,7 @@ <h3>
         <ul>
           <li><a href="https://bugs.chromium.org/p/chromium/issues/list?can=2&q=label%3AProj-VR">Chrome issues</a> (<a href="https://crbug.com/">file issue</a>)</li>
           <li><a href="https://bugzilla.mozilla.org/buglist.cgi?quicksearch=%5Bwebvr%5D">Firefox issues</a> (<a href="https://bugzilla.mozilla.org/enter_bug.cgi?product=Core&amp;component=WebVR&amp;status_whiteboard=[webvr]&amp;cc=kgilbert@mozilla.com">file issue</a>)</li>
```
[the remark this answers is not in the document]
Comment (fernandojsg) [reply developers/index.html]: done

File: developers/index.html [hunk 4]
```diff
@@ -115,6 +119,7 @@ <h3>
...
+          <li><a href="https://github.com/MozillaReality/FirefoxReality/issues">Firefox Reality</a> (<a href="https://github.com/MozillaReality/FirefoxReality/issues/new">file issue</a>)</li>
           <li><a href="http://developer.samsung.com/internet#gearvr-overview">Samsung Internet Browser for Gear VR issues</a> (<a href="https://github.com/SamsungInternet/support/issues">file issue</a>)</li>
```
[the remark this answers is not in the document]
Comment (fernandojsg) [reply developers/index.html]: done

File: index.html [hunk 5]
```diff
@@ -79,7 +79,7 @@ <h3>
                         <span class="carrot"></span>
                         <div class="device-box-header">Daydream</div>
                         <span>
-                            Works with <a href="https://play.google.com/store/apps/details?id=com.android.chrome">Chrome</a> on Daydream-ready Android devices.
+                            Works with <a href="https://play.google.com/store/apps/details?id=com.android.chrome">Chrome</a> and <a href="https://play.google.com/store/apps/details?id=org.mozilla.vrbrowser">Firefox Reality</a> on Daydream-ready Android devices.
```
[the remark this answers is not in the document]
Comment (fernandojsg) [reply index.html]: I'll leave it as currently it's available on all the daydream devices, not just standalone. We could modify it later if needed

File: index.html [hunk 6]
```diff
@@ -124,6 +124,14 @@ <h3>
                             On macOS, you can use <a href="https://nightly.mozilla.org/">Firefox Nightly</a>.
                         </span>
                     </div>
+                    <div class="device-box" onclick="this.classList.toggle('open');">
+                        <img src="images/HTC%20Vive.png" alt="Vive Port">
```
[the remark this answers is not in the document]
Comment (fernandojsg) [reply index.html]: done

File: index.html [hunk 7]
```diff
@@ -124,6 +124,14 @@ <h3>
...
+                        <span class="carrot"></span>
+                        <div class="device-box-header">Vive Port</div>
```
[the remark this answers is not in the document]
Comment (fernandojsg) [reply index.html]: done

**Comment by fernandojsg** [comment]
@cvan thanks for the suggestions!

File: developers/index.html [hunk 8]
```diff
@@ -44,12 +44,12 @@ <h3>
...
-          <a href="http://developer.samsung.com/technical-doc/view.do?v=T000000202">Samsung Internet for Gear VR</a>,
-          <a href="https://www.supermedium.com">Supermedium for Windows</a>, and
+          The WebVR API is currently available in <a href="https://webvr.rocks/firefox">Firefox for Windows</a>; <a href="https://mixedreality.mozilla.org/firefox-reality/">Firefox Reality for Oculus Go, Daydream, and VIVEPORT</a>; 
+          <a href="https://www.google.com/chrome/browser/mobile/index.html">Chrome 56+ for Android</a> (<a href="https://webvr.rocks/chrome_for_android#what_is_the_webvr_origin_trial">Origin Trial</a>);
+          <a href="../get-chrome/">Chrome Canary for Windows</a>;
+          <a href="https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webvr/">Microsoft Edge for Windows</a>;
+          <a href="http://developer.samsung.com/technical-doc/view.do?v=T000000202">Samsung Internet for Gear VR</a>;
+          <a href="https://www.supermedium.com">Supermedium for Windows</a>; and
           <a href="https://nightly.mozilla.org/">Firefox Nightly builds</a> on macOS.
```
Comment (cvan) [inline developers/index.html]: let's change `Firefox Nightly builds on macOS` to `Firefox Nightly builds for macOS` for consistency

File: developers/index.html [hunk 9]
```diff
@@ -59,7 +59,10 @@ <h3>
             <a href="https://www.google.com/chrome/browser/mobile/index.html" class="link-important">Download Chrome for Android</a> (<a href="https://webvr.rocks/chrome_for_android#setup">read instructions</a>)
           </li>
           <li>
-            <a href="https://webvr.rocks/firefox" class="link-important">Download Firefox</a> and <a href="https://nightly.mozilla.org/">Firefox Nightly builds (Vive on macOS)</a> (<a href="https://webvr.rocks/firefox#setup">read instructions</a>)
+            <a href="https://webvr.rocks/firefox" class="link-important">Download Firefox</a> and <a href="https://nightly.mozilla.org/">Firefox Nightly builds (VIVE on macOS)</a> (<a href="https://webvr.rocks/firefox#setup">read instructions</a>)
```
Comment (cvan) [inline developers/index.html]: :+1: thanks for uppercasing VIVE

**Comment by cvan** [comment]
thanks!

**Pull request merged**